great_expectations/datasource/fluent/interfaces.py

```diff
@@ -231,6 +226,11 @@ def build_batch_request(
             """One must implement "build_batch_request" on a DataAsset subclass."""
         )
 
+    def get_batch_list_from_batch_request(
```

Comment:
Changing order for clarity (`BatchRequest` goes first, getting `Batch` samples, referenced by it, goes second.